### PR TITLE
P/brent/device avatar

### PR DESCRIPTION
**Pull request by brentpicasso**
Adds device avatar to API

### CODE REVIEW

File: podium_api/devices.py [hunk 1]
```diff
@@ -27,6 +30,12 @@ def make_device_update(
     Kwargs:
         name(str): Name of the device.
 
+        private(int): 1 if the device is private to the creating user
```
Comment (DexerBR) [inline podium_api/devices.py:33]: How about accepting the conventional boolean (True | False) instead of 1 | 0?

File: podium_api/types/device.py [hunk 2]
```diff
@@ -17,12 +17,13 @@ class PodiumDevice(object):
...
     """
 
-    def __init__(self, device_id, uri, serial, name, private):
+    def __init__(self, device_id, uri, serial, name, private, avatar_url):
         self.device_id = device_id
         self.uri = uri
         self.serial = serial
         self.name = name
         self.private = private
+        self.avatar_url = avatar_url
```
Comment (DexerBR) [inline podium_api/types/device.py:26]: I suppose the avatar name can be obtained from the url, correct?

**Review by DexerBR — COMMENTED** [review]
Just a minor comment.

File: podium_api/devices.py (hunk 1 above)
Comment (brentpicasso) [reply podium_api/devices.py]: The server side code has trouble accepting JSON booleans (Ruby on Rails) so the API uses integers to indicate true / false. 

However, your suggestion is good, so I made the python API use boolean, and converts them to and from integer as needed.  

Update code to match this.

**Review by DexerBR — APPROVED** [review]
LGTM!

**Pull request merged**